projects/vdk-plugins/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Copyright 2021 VMware, Inc.
```

Comment:
This seems wrong. __init__.py does not have place here. vdk-plugins is not a python module. It's just 
a directory

Reply:
seems these two got added accidentally, I'll get rid of them